sklego/pandas.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import pandas as pd
+from scipy.ndimage.interpolation import shift
+
+
+def _as_list(val):
```

Comment:
the `as_list` method can actually be kind of useful but feels too general to be placed here. maybe move it to `common.py`. @MBrouns, thoughts?

Reply:
might be nicer to make sure we handle tuples here as well.

Reply:
Yes, I agree with moving it to common

Reply:
I'll modify the function a bit to be more generally usable.